inc/saiport.h

```diff
@@ -1214,6 +1217,15 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_PTP_MODE,
 
+    /**
+     * @brief Serdes object ID for the port
+     *
+     * @type sai_object_id_t
+     * @flags READ_ONLY
```

Comment:
Shouldn't this be write-able ? If this is read-only, how would the caller supply the serdes settings ?

Reply:
No. The SAI_PORT_SERDES object has attribute SAI_PORT_SERDES_ATTRIBUTE_PORT_ID. This attribute is for getting the reverse mapping. much like what we have for getting vlan_member_list in vlan object

Reply:
The way the serdes object came into being, serdes object is an attribute of port and not the other way round. So if you were to have a reverse-mapping, it should be in the serdes object (READ_ONLY port obj) instead of here.